qb_upload_limit_per_day.py

```diff
@@ -3,14 +3,14 @@
 import time
 import datetime
 import schedule
-
+from os.path import exists
```

Comment:
```suggestion

from os.path import exists
```

I always separate the import of modules and functions. Its just my way of writing nothing wrong with ur code.